packages/api/test/__fixtures__/sdk/operationid-quirks/index.d.ts

```diff
@@ -1,86 +0,0 @@
-import Oas from 'oas';
```

Comment:
Because this `operationid-quirks` SDK fixture is TS-first, `index.d.ts` doesn't exist anymore because we don't generate `.d.ts` files for TS output (because TS can use the types in the `.ts` files instead).